test/controllers/consolidations/AssociateDucrSummaryControllerSpec.scala

```diff
@@ -26,52 +25,29 @@ import forms.{AssociateDucr, Choice, MucrOptions}
 import org.mockito.ArgumentMatchers.{any, anyString, eq => meq}
 import org.mockito.Mockito._
 import org.mockito.{InOrder, Mockito}
+import org.scalatest.MustMatchers
 import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.{MustMatchers, WordSpec}
-import org.scalatestplus.play.guice.GuiceOneAppPerSuite
-import play.api.Application
-import play.api.i18n.{Messages, MessagesApi}
-import play.api.inject.bind
-import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.json.Json
-import play.api.test.FakeRequest
 import play.api.test.Helpers._
-import services.{CustomsCacheService, SubmissionService}
-import uk.gov.hmrc.auth.core.AuthConnector
 import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
 
 import scala.concurrent.Future
 
 class AssociateDucrSummaryControllerSpec
-    extends WordSpec with GuiceOneAppPerSuite with MockAuthConnector with ScalaFutures with MustMatchers
-    with URIHelper {
+    extends MovementBaseSpec with MockAuthConnector with ScalaFutures with MustMatchers with URIHelper {
 
   private val uri = uriWithContextPath("/associate-ducr-summary")
 
-  private val submissionServiceMock = buildSubmissionServiceMock
-  private val customsCacheServiceMock = buildCustomsCacheServiceMock
-  override lazy val app: Application =
-    GuiceApplicationBuilder()
-      .overrides(
-        bind[AuthConnector].to(authConnectorMock),
-        bind[SubmissionService].to(submissionServiceMock),
-        bind[CustomsCacheService].to(customsCacheServiceMock)
-      )
-      .build()
-
-  private val messagesApi: MessagesApi = app.injector.instanceOf[MessagesApi]
-  private implicit val messages: Messages = messagesApi.preferred(FakeRequest())
-
   private val routingHelper = RoutingHelper(app, uri)
 
   private trait Test {
     implicit val headerCarrierMock = mock[HeaderCarrier]
-    reset(authConnectorMock, submissionServiceMock, customsCacheServiceMock)
-    when(customsCacheServiceMock.fetchAndGetEntry[Choice](any(), meq(Choice.choiceId))(any(), any(), any()))
+    reset(authConnectorMock, mockSubmissionService, mockCustomsCacheService)
```

Comment:
Why this change? Now it seems that 'mock' is a verb instead of noun.

Reply:
It was just local inline from MovementBaseSpec